### ISSUE TITLE
TypeScript build-time error: `Cannot find type definition file for 'types/cross-fetch'`

### ISSUE THREAD

**Issue by timswalling** [prismicio/prismic-javascript#133]
When attempting to build a TypeScript project that has `prismic-javascript@3.0.0` as a dependency, I get the following error:

```bash
node_modules/prismic-javascript/types/index.d.ts:1:23 - error TS2688: Cannot find type definition file for 'types/cross-fetch'.

1 /// <reference types="types/cross-fetch" />
                        ~~~~~~~~~~~~~~~~~


Found 1 error.
```

Is [the triple-slash directive in `index.d.ts`](https://github.com/prismicio/prismic-javascript/blob/3.0.0/types/index.d.ts#L1) supposed to be:

```ts
/// <reference types="cross-fetch" />
```
?

**Comment by noclat** [comment]
Same here. Please make this issue a priority, as it'll prevent any TypeScript project from building.

**Comment by lludol** [comment]
I found a dirty solution for now : 
```tsc --skipLibCheck```

**Comment by MartijnHols** [comment]
A cleaner solution may be to use https://github.com/ds300/patch-package to fix the type in your repo

**Comment by kamilobello** [comment]
Same here, another update?

**Comment by Quineone** [comment]
Same here, any process ?

**Comment by hansottowirtz** [comment]
In `package.json`, you can use `"prismic-javascript": "github:pvdstel/prismic-javascript#patch-1"` until #138 is merged.

**Comment by shyamal890** [comment]
This is still there. Hope it gets solved soon

**Comment by lihbr** [comment]
Hi everyone, sorry for the delay, I'll try to sync with the team to have this small fix published 🙂

**Comment by veloce** [comment]
Fixed in b998175b
